crates/sui-types/src/crypto.rs

```diff
@@ -965,25 +958,6 @@ impl<S: SuiSignatureInner + Sized> SuiSignature for S {
                 error: format!("{}", e),
             })
     }
-
-    fn add_to_verification_obligation_or_verify(
```

Comment:
So this function will either add to the obligation list (anything that can be batched verified) or literally verify the signature. Now that we removed it, where do we verify sigs that cannot be batched?

Reply:
This function was 100% unused, so there was no change in behaviour related to this function.
The place where we check the user's sig is here:
https://github.com/MystenLabs/sui/blob/97e96ba1cf2cd8226bc31a7aecd35b6fdd7ea37a/crates/sui-core/src/consensus_validator.rs#L72